src/utils/fields.js

```diff
@@ -11,6 +11,7 @@ import {
     FIELD_NAME,
     FIELD_TYPE,
     FIELD_SIZE,
+    FIELD_PROMOTED_BY,
```

Comment:
could you move to this the `[...]_BY` section below to keep the user fields together

```
FIELD_CREATED_BY,
FIELD_MODIFIED_BY,
FIELD_OWNED_BY,
FIELD_RESTORED_BY,
FIELD_TRASHED_BY,
```

Reply:
Yeah, np